cores/esp8266/core_esp8266_waveform.cpp

```diff
@@ -176,12 +176,6 @@ int ICACHE_RAM_ATTR stopWaveform(uint8_t pin) {
   if (!timerRunning) {
     return false;
   }
-  // If user sends in a pin >16 but <32, this will always point to a 0 bit
-  // If they send >=32, then the shift will result in 0 and it will also return false
-  uint32_t mask = 1<<pin;
```

Comment:
I'm getting a compilation error
```
/Users/xxx/.platformio/packages/framework-arduinoespressif8266/cores/esp8266/core_esp8266_waveform.cpp:179:24: error: 'mask' was not declared in this scope
   waveformToDisable |= mask;
```

Reply:
Yeah, please pull latest.  Silly mistake no my part.  Current PR works.

Reply:
@s-hadinger , I meant to check out #7231 in my other mail. Sorry, please use that one instead for your test.

This is unrelated to PWM stuff.